content/es/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -0,0 +1,751 @@
+|:------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Pending`   | El clúster de Kubernetes aceptó el Pod, pero uno o más contenedores no se configuraron ni prepararon para ejecutarse. Esto incluye el tiempo que pasa un Pod esperando ser programado, así como el tiempo dedicado a descargar imágenes de contenedores a través de la red. |
+| `Running`   | El Pod se vinculó a un nodo y se crearon todos los contenedores. Al menos un contenedor todavía se está ejecutando o está en proceso de iniciarse o reiniciarse.                                                                                                            |
+| `Succeeded` | Todos los contenedores del Pod finalizaron con éxito y no se reiniciarán.                                                                                                                                                                                                   |
+| `Failed`    | Todos los contenedores del Pod han finalizado y al menos un contenedor ha finalizado con error. Es decir, el contenedor salió con un estado distinto de cero o el sistema lo canceló.                                                                                       |
+| `Unknown`   | Por alguna razón no se pudo obtener el estado del Pod. Esta fase generalmente ocurre debido a un error en la comunicación con el nodo donde debería ejecutarse el Pod.                                                                                                      |
+
+{{< note >}}
+Cuando se está borrando un Pod, se muestra como `Terminating` por algunos
+comandos de kubectl.
+Este estado `Terminating` no es una de las fases del Pod.
+A un Pod se le garantiza un tiempo para terminar con gracia,
+cuyo valor por defecto es 30 segundos.
+Puedes utilizar el flag `--force`
+para [terminar un Pod por la fuerza](#pod-termination-forced).
+{{< /note >}}
+
+A partir de la versión 1.27 de Kubernetes, el kubelet aplica una transición de
+los Pods borrados, excepto
+por [Pods estáticos](/docs/tasks/configure-pod-container/static-pod/)
+y [Pods borrados por la fuerza](#pod-termination-forced) sin un finalizador, a
+una fase terminal
+(`Failed` o `Succeeded` dependiendo de los códigos de salida de los contenedores
+del Pod) antes de su eliminación del servidor API.
+
+Si un Nodo muere o se desconecta del resto del clúster,
+Kubernetes aplica una política para establecer la `phase` de todos los Pods
+en `Failed`.
+
+## Estados del contenedor
+
+Así como la fase del Pod en general, Kubernetes rastrea el estado de cada
+contenedor dentro de un Pod.
+Puedes
+usar [hooks del ciclo de vida de un contenedor](/docs/concepts/containers/container-lifecycle-hooks/)
```

Comment:
`content/es/docs/concepts/containers/container-lifecycle-hooks.md` page is already translated to Spanish, we could reference it here:

```suggestion
usar [hooks del ciclo de vida de un contenedor](/es/docs/concepts/containers/container-lifecycle-hooks/)
```